docs/topics/develop/patterns/asynchronous-processing.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 6
+title: Best Practices for Async Process with Momento Topics
+sidebar_label: Asynchronous Processing
+description: Learn best practices around using Momento Topics and webhooks to process data asynchronously in your event-driven architectures
+keywords:
+ - topics
+ - webhooks
+ - eda
+ - momento
+ - event-driven architecture
+ - step functions
+---
+
+# Use Momento Topics to process data asynchronously
+
+With Momento Topics, you can subscribe to messages on a Topic, as well as publish messages to a new Topic. Webhooks allow you to connect these Topics to stateless consumers, which can then process these events asynchronously. Whether this is aggregating events by `topic_id`, saving each event to a database, or using the payload to trigger a step function, webhooks give you the flexibility to process these events how you want to. 
+
+The key to asynchnously processing these events it to use _multiple_ topics. An inbound topic, which the Webhook listens to, and a single, or multiple, outbound topics which publish the processed data. For a more advanced use case, you can even send messages back to the publishing client via Topics.
+
+![Architecture](@site/static/img/topics/patterns/asynchronous-processing.png)
+
+## Pre-requisites
+1. A public facing endpoint to receive webhook events. This endpoint must accept POST requests and be able to receive inbound calls from Momento. More detail about the structure of this event is [described here.](https://docs.momentohq.com/topics/webhooks/overview#example-event)
+
+## Getting Started
+1. [Create a cache in the Momento console](https://console.gomomento.com/caches/create)
+2. [Create a webhook](/topics/webhooks/creating-a-webhook) for the cache. Assign the Webhook Destination to the public facing endpoint.
+3. Add code to the Webhook to process the incoming messages
```

Comment:
I might add a sentence to this bullet that says "For example, if your webhook is implemented as an AWS Lambda behind an API Gateway, then here is some very simple lambda example code that processes the incoming message, converts it to upper case, and then publishes it to the outbound topic:"